src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -157,6 +254,52 @@ public void testSystemCrashReportEmailContent() throws IOException {
                                        Config.getAppVersion(), error.getMessage());
         
         verifyEmail(email, Config.SUPPORT_EMAIL, subject, "/systemCrashReportEmail.html");
+        
+        ______TS("user is not logged in and the error has no message");
+        
+        error = new AssertionError();
+        email = new EmailGenerator().generateSystemErrorEmail(requestMethod, requestUserAgent, requestPath,
+                                                              requestUrl, requestParam, null, error);
+        subject = String.format(EmailType.ADMIN_SYSTEM_ERROR.getSubject(),
+                                Config.getAppVersion(), "java.lang.AssertionError");
+        modifiedContent = email.getContent().replaceAll(lastCommonLineRegex, "$1...$2");
+        email.setContent(modifiedContent);
+        
+        verifyEmail(email, Config.SUPPORT_EMAIL, subject, "/systemCrashReportEmail2.html");
```

Comment:
A minor point and you can ignore this if you want, but let's try to think of a better name than suffixing 2 behind, if possible.